pallets/liquidity-pools/src/message.rs

```diff
@@ -206,7 +239,7 @@ impl BatchMessages {
 	MaxEncodedLen,
 	Default,
 )]
-pub enum Message<BatchContent = BatchMessages> {
+pub enum Message<BatchContent = BatchMessages, ForwardContent = NonForwardMessage> {
```

Comment:
Are curious about where a message with ForwardContent != NonForwardMessage is created. I think you always create instances with `NonForwardMessage`, right?

Reply:
Yes. I suppose this can be reduced then? Would like to merge this PR as is assuming CI is green. Will clean up in a follow up PR. Thanks for spotting this!